code/javascript/assignment#1/string.js

```diff
@@ -1,7 +1,7 @@
 //assignemnet #1 using my name
 console.log("whats your name")
 console.log("danny uche")
-// now i was just trying to remember my name
+// now i was just trying to remember my name now
 
 const name= "Danny U"
```

Comment:
name' is deprecated.ts(6385)
lib.dom.d.ts(27433, 5): The declaration was marked as deprecated here. next time , use capital letter like this below.

const Name= "Danny U"
console.log(Name.length)
console.log("Hello my name is "+ Name)
console.log("When my friends see me they shout" + Name + "!")

let newName=Name.toUpperCase()
console.log("When my friend see me, they shout" + " " + newName + "!" ) , but the code you provided worked as expected so next time use Capital letter for the first word.